providers/tests/system/amazon/aws/example_mwaa.py

```diff
@@ -0,0 +1,88 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow.models.baseoperator import chain
+from airflow.models.dag import DAG
+from airflow.providers.amazon.aws.operators.mwaa import MwaaTriggerDagRunOperator
+
+from providers.tests.system.amazon.aws.utils import ENV_ID_KEY, SystemTestContextBuilder
+
+DAG_ID = "example_mwaa"
+
+# Externally fetched variables:
+EXISTING_ENVIRONMENT_NAME_KEY = "ENVIRONMENT_NAME"
+EXISTING_DAG_ID_KEY = "DAG_ID"
+
+
+sys_test_context_task = (
+    SystemTestContextBuilder()
+    # NOTE: Creating a functional MWAA environment is time-consuming and requires
+    # manually creating and configuring an S3 bucket for DAG storage and a VPC with
+    # private subnets which is out of scope for this demo. To simplify this demo and
+    # make it run in a reasonable time, follow these steps in the AWS Console to create
+    # a new MWAA environment with default configuration:
+    # 1. Create an S3 bucket and upload your DAGs to a 'dags' directory
+    # 2. Navigate to the MWAA console
+    # 3. Create an environment, making sure to use the S3 bucket from the previous step
+    # 4. Use the default VPC/network settings (or create a new one using this guide:
+    # https://docs.aws.amazon.com/mwaa/latest/userguide/vpc-create.html)
+    # 5. Select Public network for web server access and click through to creation
+    .add_variable(EXISTING_ENVIRONMENT_NAME_KEY)
+    .add_variable(EXISTING_DAG_ID_KEY)
+    .build()
+)
+
+with DAG(
+    dag_id=DAG_ID,
+    schedule="@once",
+    start_date=datetime(2021, 1, 1),
+    tags=["example"],
+    catchup=False,
+) as dag:
+    test_context = sys_test_context_task()
+    env_id = test_context[ENV_ID_KEY]
+    env_name = test_context[EXISTING_ENVIRONMENT_NAME_KEY]
+    trigger_dag_id = test_context[EXISTING_DAG_ID_KEY]
+
+    # [START howto_operator_mwaa_trigger_dag_run]
+    trigger_dag_run = MwaaTriggerDagRunOperator(
+        task_id="trigger_dag_run",
+        env_name=env_name,
+        trigger_dag_id=trigger_dag_id,
+    )
```

Comment:
Is the use case here that you running the dag in environment A yet want to trigger another DAG in environment B?

Reply:
Yes exactly, assuming at least environment B is running on AWS MWAA. And as for the hook, it would let you perform any API action on environment B.

Reply:
Please update docs to explain the difference with the regular tigger dag run operator

Reply:
Done!